themes/Default/overlay.mumblelay

```diff
@@ -1,5 +1,5 @@
 [overlay]
-version=1.4.0
+version=1.5
```

Comment:
Instead of hard-coding this here, we should instead let cmake configure this file to fill in the current version automatically